Lib/concurrent/futures/process.py

```diff
@@ -855,3 +856,29 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
+
+    def terminate_workers(self, signal=signal.SIGINT):
```

Comment:
SIGTERM should be preferred over SIGINT as a default termination. AFAIR, SIGINT is roughly equivalent to hit "Ctrl-C" but SIGTERM is the programatic way to ask a process to terminate. However, considering the usage of this function, I also see value in using SIGINT.

Reply:
Makes sense to me. Will swap.